src/shared-components/button/constants.js

```diff
@@ -0,0 +1,25 @@
+import { COLORS } from '../../constants';
+
+// TODO: potentially break out these pairings to the color constants file
+export const textColorsAssociatedWithColors = {
```

Comment:
wasnt sure if we wanted this in our actual constants/colors file since its kinda pointed to how i was using it. Maybe there's a better way to connect colors together as related.